constraints/miden-vm/rpo.air

```diff
@@ -0,0 +1,80 @@
+mod RPOAir
+
+### Constants and periodic columns ################################################################
+const MDS = [
+    [7, 23, 8, 26, 13, 10, 9, 7, 6, 22, 21, 8],
+    [8, 7, 23, 8, 26, 13, 10, 9, 7, 6, 22, 21],
+    [21, 8, 7, 23, 8, 26, 13, 10, 9, 7, 6, 22],
+    [22, 21, 8, 7, 23, 8, 26, 13, 10, 9, 7, 6],
+    [6, 22, 21, 8, 7, 23, 8, 26, 13, 10, 9, 7],
+    [7, 6, 22, 21, 8, 7, 23, 8, 26, 13, 10, 9],
+    [9, 7, 6, 22, 21, 8, 7, 23, 8, 26, 13, 10],
+    [10, 9, 7, 6, 22, 21, 8, 7, 23, 8, 26, 13],
+    [13, 10, 9, 7, 6, 22, 21, 8, 7, 23, 8, 26],
+    [26, 13, 10, 9, 7, 6, 22, 21, 8, 7, 23, 8],
+    [8, 26, 13, 10, 9, 7, 6, 22, 21, 8, 7, 23],
+    [23, 8, 26, 13, 10, 9, 7, 6, 22, 21, 8, 7]
+]
+
+periodic_columns:
```

Comment:
let's add a comment to explain the round constants and how they are laid out here